client/tests/simulator/mod.rs

```diff
@@ -0,0 +1,9 @@
+#![cfg(feature = "tpm-simulator-tests")]
+
+// Please see "simulator_tests.rs" for details.
+//
+// Add at least one test module file for each chapter in TPM command spec.
+// If there are many tests for a single command, it is recommended to split into command specific files.
+// For example: <chaptername>_<commandname>.rs
```

Comment:
My recommendation would be that they create further submodules. E.g.

*   `<chaptername>/mod.rs`
*   `<chaptername>/<commandname_1>.rs`
*   `<chaptername>/<commandname_2>.rs`

Reply:
The only place where submodule resolution is weird is directly under the `tests` directory, now that we're under a normal module things should work in a pretty expected manner.

Reply:
I have updated the text recommendation. Fixed.